packages/react-native-win32/src/Libraries/Image/ImageViewNativeComponent.win32.js

```diff
@@ -42,13 +43,16 @@ type NativeProps = $ReadOnly<{|
 let ImageViewNativeComponent;
 
 if (global.RN$Bridgeless) {
-  ImageViewNativeComponent = codegenNativeComponent<NativeProps>(
-    'RCTImage', // [Win32] Rename RCTImageView to RCTImage
-  );
+  ReactNativeViewConfigRegistry.register('RCTImage', () => {
+    // [Win32] Rename RCTImageView to RCTImage
+    return ImageViewViewConfig;
+  });
+  ImageViewNativeComponent = 'RCTImage'; // [Win32] Rename RCTImageView to RCTImage
 } else {
   ImageViewNativeComponent = requireNativeComponent<NativeProps>(
     'RCTImage', // [Win32] Rename RCTImageView to RCTImage
```

Comment:
Hmm. in the source I see RCTImageView: https://github.com/facebook/react-native/blob/master/Libraries/Image/ImageViewNativeComponent.js#L51

Should these comments say "Rename RCTImage to RCTImageView"?

Reply:
Depends if its documenting what the win32 diff is compared to master, or if its documenting a change we need to make in native.  In this case it looks like its trying to document the diff.

Reply:
Yeah, I've been commenting what the change is instead of as a TODO.